### PR TITLE
Move OC/Core completely to PSR-4

### PR DESCRIPTION
**Pull request by LukasReschke**
cc @rullzer @MorrisJobke

### CODE REVIEW

File: lib/autoloader.php [hunk 1]
```diff
@@ -110,7 +110,6 @@ public function findClass($class) {
 				$split[1] = strtolower($split[1]);
 
 				if ($split[1] === 'core') {
-					$paths[] = \OC::$SERVERROOT . '/core/' . strtolower(str_replace('\\', '/', $split[2])) . '.php';
```
Comment (MorrisJobke) [inline lib/autoloader.php]: Could you add a message, that this is skipped because it is already covered by PSR-4

Comment (LukasReschke) [inline lib/autoloader.php]: Done :)

**Comment by MorrisJobke** [comment]
Tested and works 👍

**Comment by rullzer** [comment]
:+1:

**Pull request merged**